components/ReplyToMemberInvitationCard.js

```diff
@@ -62,6 +63,10 @@ const ReplyToMemberInvitationCard = ({ invitation, isSelected, refetchLoggedInUs
   const hasReplied = data && typeof data.replyToMemberInvitation !== 'undefined';
 
   const buildReplyToInvitation = accept => async () => {
+    if (accept && redirectOnAccept) {
+      await Router.pushRoute(`/${invitation.collective.slug}`);
+    }
```

Comment:
The redirects needs to happen after we accept the invitation (`sendReplyToInvitation`). Ideally, it should also happen after `refetchLoggedInUser` to make sure we refresh the permissions before loading the page.

Reply:
Done!! Should I also change the commit message to match the new PR description ?